core/src/test/java/org/apache/iceberg/rest/responses/TestLoadTableResponse.java

```diff
@@ -0,0 +1,371 @@
+        "    \"current-schema-id\":7," +
+        "    \"schemas\":[{" +
+        "      \"type\":\"struct\",\"schema-id\":7,\"fields\":[" +
+        "        {\"id\":1,\"name\":\"x\",\"required\":true,\"type\":\"long\"}," +
+        "        {\"id\":2,\"name\":\"y\",\"required\":true,\"type\":\"long\",\"doc\":\"comment\"}," +
+        "        {\"id\":3,\"name\":\"z\",\"required\":true,\"type\":\"long\"}" +
+        "      ]},{" +
+        "      \"type\":\"struct\",\"schema-id\":6,\"fields\":[" +
+        "        {\"id\":10,\"name\":\"x\",\"required\":true,\"type\":\"string\"}]" +
+        "    }]," +
+        "  \"default-spec-id\":5," +
+        "  \"partition-specs\":[{\"spec-id\":5,\"fields\":[]}]," +
+        "  \"last-partition-id\":999," +
+        "  \"default-sort-order-id\":3," +
+        "  \"sort-orders\":[" +
+        "    {\"order-id\":3,\"fields\":[" +
+        "      {\"transform\":\"identity\",\"source-id\":2,\"direction\":\"asc\",\"null-order\":\"nulls-first\"}," +
+        "      {\"transform\":\"bucket[4]\",\"source-id\":3,\"direction\":\"desc\",\"null-order\":\"nulls-last\"}]}]," +
+        "  \"properties\":{\"property\":\"value\"}," +
+        "  \"current-snapshot-id\":" + metaV2.currentSnapshot().snapshotId() + "," +
+        "  \"refs\":{" +
+        "    \"main\":{" +
+        "      \"snapshot-id\":" + metaV2.ref("main").snapshotId() + "," +
+        "      \"type\":\"branch\"}," +
+        "    \"previous\":{" +
+        "      \"snapshot-id\":" + metaV2.ref("previous").snapshotId() + "," +
+        "      \"type\":\"tag\"}," +
+        "    \"test\":{" +
+        "      \"snapshot-id\":" + metaV2.ref("test").snapshotId() + "," +
+        "     \"type\":\"branch\"}" +
+        "  }," +
+        "  \"snapshots\":[{" +
+        "    \"snapshot-id\":" + metaV2.snapshots().get(0).snapshotId() + "," +
+        "    \"timestamp-ms\":" + metaV2.snapshots().get(0).timestampMillis() + "," +
+        "    \"manifests\":[" +
+        "      \"/tmp/manifest.1.avro\"" +
+        "    ]},{" +
+        "    \"snapshot-id\":" + metaV2.snapshots().get(1).snapshotId() + "," +
+        "    \"parent-snapshot-id\":" + metaV2.snapshots().get(1).parentId() + "," +
+        "    \"timestamp-ms\":" + metaV2.snapshots().get(1).timestampMillis() + "," +
+        "    \"manifests\":[" +
+        "      \"/tmp/manifest.2.avro\"" +
+        "    ],\"schema-id\":7" +
+        "  }]," +
+        "  \"snapshot-log\":[{" +
+        "      \"timestamp-ms\":" + previousSnapshot.timestampMillis() + "," +
+        "      \"snapshot-id\":" + previousSnapshot.snapshotId() +
+        "    },{" +
+        "      \"timestamp-ms\":" + currentSnapshot.timestampMillis() + "," +
+        "      \"snapshot-id\":" + currentSnapshot.snapshotId() +
+        "  }]," +
+        "  \"metadata-log\":[]}," +
+        "  \"config\":{\"foo\":\"bar\"}}";
+
+    assertRoundTripSerializesEquallyFrom(fullExampleInstanceJson.replaceAll("\\s", ""), respV2);
+
+  }
+
+
```

Comment:
Style: only one empty line is needed between methods.

Reply:
This has been updated